### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

**Comment by briankassouf** [comment]
You can view the readme formatted here: https://github.com/Imgur/incus/blob/readme-update/README.md

File: README.md [hunk 1]
```diff
@@ -1,21 +1,348 @@
...
+$('#button').on('click', function() {
+    socket.MessageUser('Event', 'UID', data); 
+    socket.MessageAll('Event1', data);
+    socket.MessagePage('Event2', '/page/path', data);
+});
+```
+
+### Application to a web browser
+
+UID is a unique identifier for the user. It can be anywhere from an auto incremented ID to something more private such as a session id or OAuth token.
```
Comment (jacob-pascual) [inline README.md]: suggestion: something like "If messages to users should be private from other users, then you should consider the UID a shared secret between the user and the server" - suggests something similar to the scheme we have.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,348 @@
...
+* if user and page are both unset the message object will be sent to all users
+* if both user and page are set the message object will be sent to that user on that page
+* if just user is set, the message object will be sent to all sockets owned by the user identified by UID
+* if just page is set, the message object will be sent to all sockets whose page matches the page identifier
+
+
+### Push notifications 
+To send push notifications from your app, you need to push a json formated string to a **Redis list**. The list key is configurable but defaults to `Incus_Queue`
+
+Android and iOS have slightly differnt schemas for sending push notifications.
```
Comment (jacob-pascual) [inline README.md]: `different`

File: README.md [hunk 3]
```diff
@@ -1,21 +1,348 @@
...
+Start an instance of Redis:
+
+```Shell
+docker run -d --name incusredis redis
+```
+
+Start Incus:
+
+```Shell
+docker run -d --link incusredis:redis --name incus \
```
Comment (jacob-pascual) [inline README.md]: `-e REDIS_ENABLED=true`

**Comment by jacob-pascual** [comment]
:+1:

**Comment by jasdev** [comment]
Looks good :+1:

**Pull request merged**